test/client/fixtures/runner/automation/regression-test.js

```diff
@@ -775,4 +775,33 @@ $(document).ready(function () {
             });
         });
     });
+
+    asyncTest('T235186 - Focus event handlers don\'t call for iframe\'s contenteditable body', function () {
+        var focusRaised = false,
+            $iFrame     = $('<iframe></iframe>')
+                .width(500)
+                .height(500)
+                .attr('src', window.QUnitGlobals.getResourceUrl('../../../data/focus-blur-change/iframe.html'))
+                .addClass(TEST_ELEMENT_CLASS)
+                .appendTo('body');
+
+        $iFrame.load(function () {
+            var $iFrameBody = $($iFrame[0].contentWindow.document.body);
+            $iFrameBody.attr('contenteditable', true);
+
+            $iFrameBody.bind('focus', function () {
+                focusRaised = true;
```

Comment:
I'd prefer to use the `focusRaisedCount` counter here because in fact we check focus handler raising count in the test